### PR TITLE
Fix more PHP 8 undefined array index errors

### PR DESCRIPTION
**Pull request by ausi**
Some more ☺️

### CODE REVIEW

File: core-bundle/src/Resources/contao/library/Contao/Controller.php [hunk 1]
```diff
@@ -1506,7 +1506,7 @@ public static function addImageToTemplate($template, array $rowData, $maxWidth =
 				Metadata::VALUE_ALT => $rowData['alt'] ?? '',
 				Metadata::VALUE_TITLE => $rowData['imageTitle'] ?? '',
 				Metadata::VALUE_URL => self::replaceInsertTags($rowData['imageUrl'] ?? ''),
-				'linkTitle' => $rowData['linkTitle'] ?: '',
+				'linkTitle' => $rowData['linkTitle'] ?? '',
```
Comment (ausi) [inline core-bundle/src/Resources/contao/library/Contao/Controller.php]: @m-vo this correct?

File: core-bundle/src/Resources/contao/elements/ContentAccordion.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ protected function compile()
 				->applyLegacyTemplateData($this->Template, $this->imagemargin, $this->floating);
 		}
 
-		$classes = StringUtil::deserialize($this->mooClasses);
+		$classes = StringUtil::deserialize($this->mooClasses, true) + array('', '');
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/elements/ContentAccordion.php]: Isn't `array('', '')` a behavior change compared to `array(null, null)`?

File: core-bundle/src/Resources/contao/library/Contao/Controller.php (hunk 1 above)
Comment (m-vo) [reply core-bundle/src/Resources/contao/library/Contao/Controller.php]: Probably. In theory someone could have set `linkTitle` to `false` or `0` which would have worked in the old code but would now result in an error.

 If we wanted to be extra safe, we could for instance do one of these:

```suggestion
				'linkTitle' => ($rowData['linkTitle'] ?? null) ?: '',
```

```suggestion
				'linkTitle' => (string)($rowData['linkTitle'] ?? ''),
```

File: core-bundle/src/Resources/contao/elements/ContentAccordion.php (hunk 2 above)
Comment (ausi) [reply core-bundle/src/Resources/contao/elements/ContentAccordion.php]: Fixed in 93139b85368a3aa735f527f91f9dba3be7353084

**Review by m-vo — APPROVED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**